app/components/september-challenge/modal.tsx

```diff
@@ -0,0 +1,146 @@
+import * as React from "react"
+import { Linking, View } from "react-native"
+import Modal from "react-native-modal"
+
+import { useI18nContext } from "@app/i18n/i18n-react"
+import { makeStyles, useTheme, Text } from "@rneui/themed"
+
+import { GaloyIconButton } from "../atomic/galoy-icon-button"
+import { GaloyToast } from "../galoy-toast"
+import { GaloyIcon } from "../atomic/galoy-icon"
+import { useCirclesQuery } from "@app/graphql/generated"
+
+const CHALLENGE_PAGE = "blink.sv/circles"
+const CHALLENGE_PAGE_URL = "https://blink.sv/circles"
+
+type Props = {
+  isVisible: boolean
+  setIsVisible: (isVisible: boolean) => void
+}
+
+export const SeptemberChallengeModal: React.FC<Props> = ({ isVisible, setIsVisible }) => {
+  const { LL } = useI18nContext()
+
+  const {
+    theme: { colors },
+  } = useTheme()
+  const styles = useStyles()
+
+  const { data } = useCirclesQuery({
+    fetchPolicy: "network-only",
+  })
+
+  const isLonely =
+    data?.me?.defaultAccount.welcomeProfile === null ||
+    data?.me?.defaultAccount.welcomeProfile?.innerCircleAllTimeCount === 0
+  const innerCircle =
+    (data?.me?.defaultAccount.welcomeProfile?.innerCircleThisMonthCount || 0) + "/21"
+  const rank = data?.me?.defaultAccount.welcomeProfile?.thisMonthRank
+    ? `#${data?.me?.defaultAccount.welcomeProfile?.thisMonthRank}`
+    : "-"
+
+  const acknowledgeModal = () => {
+    setIsVisible(false)
+  }
+
+  return (
+    <Modal
+      isVisible={isVisible}
+      backdropOpacity={0.8}
+      backdropColor={colors.white}
+      onBackdropPress={acknowledgeModal}
```

Comment:
There is a flicker in this modal sometimes when you close it. I think it can be fixed by adding `backdropTransitionOutTiming={0}` to the modal props.